src/main/webapp/i18n/de/competency.json

```diff
@@ -5,7 +5,7 @@
             "deleted": "Kompetenz gelöscht",
             "updated": "Kompetenz bearbeitet",
             "competencyButton": "Kompetenzen",
-            "title": "Kompetenzen",
+            "title": "Titel",
```

Comment:
If I'm not mistaken, this is also used in the `competency-form.component.html` and `competency-management.component.html`. Are you sure this is intentional?

Reply:
This is actually an intentional "bugfix". See the screenshots from the current system - Imo. "Title" is much more accurate than "Competencies" which seems like an oversight.
![image](https://github.com/ls1intum/Artemis/assets/118574504/25fc3634-c90b-4994-b4d9-f25515e83f0c)
![image](https://github.com/ls1intum/Artemis/assets/118574504/172cd4c4-7a2c-48ab-ad52-6d2852521396)